app/resources/numismatics/issues/numismatics/issue.rb

```diff
@@ -55,8 +55,6 @@ class Issue < Resource
     attribute :depositor
     attribute :identifier
     attribute :replaces
-    attribute :rights_statement
-    attribute :rights_note, Valkyrie::Types::Set
```

Comment:
I don't think it's a good idea to remove these attributes completely. Maybe choose an appropriate default and just hide them on the edit page.

Reply:
So we should keep a default no copyright in the show page and hide them from the edit and the new form. Thanks! 
![Screen Shot 2020-02-19 at 4 46 16 PM](https://user-images.githubusercontent.com/9905193/74879554-602ba080-5337-11ea-9d23-88e52e30e775.png)